tests/test_downloader_handler_twisted_http11.py

```diff
@@ -0,0 +1,78 @@
+"""Tests for scrapy.core.downloader.handlers.http11.HTTP11DownloadHandler."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from scrapy.core.downloader.handlers.http11 import HTTP11DownloadHandler
+from tests.test_downloader_handlers_http_base import (
+    TestHttp11Base,
+    TestHttpMockServerBase,
+    TestHttpProxyBase,
+    TestHttps11Base,
+    TestHttpsCustomCiphersBase,
+    TestHttpsInvalidDNSIdBase,
+    TestHttpsInvalidDNSPatternBase,
+    TestHttpsWrongHostnameBase,
+    TestSimpleHttpsBase,
+)
+
+if TYPE_CHECKING:
+    from scrapy.core.downloader.handlers import DownloadHandlerProtocol
+
+
+DH = HTTP11DownloadHandler
+
+
+class TestHttp11(TestHttp11Base):
+    @property
+    def download_handler_cls(self) -> type[DownloadHandlerProtocol]:
+        return DH
```

Comment:
Looking at the amount of repetition, I wonder if a mixin class to implement the property would be a cleaner option than a global variable.

Reply:
duh